beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/monitoring/TimedTaggableAspect.java

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
This still needs a license header - it would be copyright Expedia and Pivotal and then under the Apache license.

Reply:
Oh really? Thanks for the heads up, shall I do this manually then? Or is there a way to add this to the mycila plugin?

Will create a separate PR for this file, cheers.

Reply:
Yeah, simplest is to do it manually and then tell Mycila not to overwrite it (that bit you've already done).